lib/ui/src/components/sidebar/SearchResults.tsx

```diff
@@ -24,6 +24,16 @@ const ResultRow = styled.li<{ isHighlighted: boolean }>(({ theme, isHighlighted
   cursor: 'pointer',
 }));
 
+const NoResults = styled.div(({ theme }) => ({
+  marginTop: 20,
+  textAlign: 'center',
+  fontSize: `${theme.typography.size.s2}px`,
```

Comment:
```suggestion
  fontSize: `${theme.typography.size.s2 - 1}px`,
  lineHeight: `18px`,
```

Aligns with the rest of the sizing.